compound.py

```diff
@@ -168,6 +168,9 @@ def update_bot_order_volumes(
     )
 
     rounddigits = get_round_digits(thebot["pairs"][0])
+    logger.debug(
```

Comment:
Is dit stukje nog nodig? Denk dat je, zelfs in debug, er weinig aan hebt? En anders zou ik het samenvoegen met de factor berekening even hieronder zodat je daar een logging als 'Number if digits is x resulting in factor y' krijgt.

Reply:
Dat is wel een goed punt.
Done